src/darwin/Framework/CHIPTests/MTRDeviceTests.m

```diff
@@ -2605,6 +2606,166 @@ - (void)test028_TimeZoneAndDST
 #endif // MTR_ENABLE_PROVISIONAL
 }
 
+/**
+ * Given a path relative to the Matter root, create an absolute path to the file.
+ */
+- (NSString *)absolutePathFor:(NSString *)matterRootRelativePath
+{
+    // Find the right absolute path to our file.  PWD should
+    // point to our src/darwin/Framework.
+    NSString * pwd = [[NSProcessInfo processInfo] environment][@"PWD"];
+    NSMutableArray * pathComponents = [[NSMutableArray alloc] init];
+    [pathComponents addObject:[pwd substringToIndex:(pwd.length - @"src/darwin/Framework".length)]];
+    [pathComponents addObjectsFromArray:[matterRootRelativePath pathComponents]];
+    return [NSString pathWithComponents:pathComponents];
+}
+
+/**
+ * Create a task given a path relative to the Matter root.
+ */
+- (NSTask *)createTaskForPath:(NSString *)path
```

Comment:
This _definitely_ needs to be conditioned on "not on iOS" bits.

Reply:
forgot to remove these. not needed anymore. removed.